autoload/ensime.vim

```diff
@@ -1,5 +1,6 @@
 if !has('nvim')
-    execute 'silent! py3file' fnameescape(expand('<sfile>:p').'.py')
+    execute 'py3file' fnameescape(expand('<sfile>:p').'.py')
```

Comment:
I think it's better not to hide the output here. Makes debugging a bit easier.

Reply:
Ok. Remove commented line then.